samples/kotlin_compose/src/main/java/io/customer/android/sample/kotlin_compose/ui/login/Login.kt

```diff
@@ -39,6 +42,13 @@ fun LoginRoute(
 ) {
     val state = loginViewModel.uiState.collectAsState()
 
+    TrackScreenLifecycle(
+        lifecycleOwner = LocalLifecycleOwner.current,
+        onScreenEnter = {
+            CustomerIO.instance().screen("Login")
```

Comment:
> Are you suggesting using automatic screen tracking instead of manual?

Yes. Because `autoScreenTracking` is not going to work with compose. And if we use the switch for `autoScreenTracking` only, that will be of no use here.

I also had confusion around this flag and asked this [on slack](https://customerio.slack.com/archives/C05C2HF5VB2/p1686301521615749) already ([similar discussion](https://customerio.slack.com/archives/C05C2HF5VB2/p1686328504860039) you might be interested in). My current understanding is, for native apps where auto screen tracking is available, we can bind this with `autoScreenTracking` in SDKs. For wrappers and SDKs where `autoScreenTracking` can not work, we can use flag for manual screen tracking. Since the flag only says Screen Tracking (and doesn't include auto), I think it isn't misleading. But yes, having different implementation may be confusing for some people.